tests/unit/commands/local/invoke/test_cli.py

```diff
@@ -46,22 +46,13 @@ def setUp(self):
         self.invoke_image = ("amazon/aws-sam-cli-emulation-image-python3.6",)
         self.hook_package_id = None
 
-    @patch("samcli.commands.local.cli_common.invoke_context.InvokeContext")
-    @patch("samcli.commands.local.invoke.cli._get_event")
-    def test_cli_must_setup_context_and_invoke(self, get_event_mock, InvokeContextMock):
-        event_data = "data"
-        get_event_mock.return_value = event_data
-
-        ctx_mock = Mock()
-        ctx_mock.region = self.region_name
-        ctx_mock.profile = self.profile
-
-        # Mock the __enter__ method to return a object inside a context manager
-        context_mock = Mock()
-        InvokeContextMock.return_value.__enter__.return_value = context_mock
+        self.ctx_mock = Mock()
+        self.ctx_mock.region = self.region_name
+        self.ctx_mock.profile = self.profile
 
+    def call_cli(self):
```

Comment:
I originally modified this file by including a parameter for `skip_prepare_iac`. While I was in this file, I moved the CLI invoke call to a helper function to reduce repeat code. I then realised that I didn't need to add `skip_prepare_iac`, but I had already made the help function improvements. That is why this file is modified.